docs/HeliumParamSpecs.md

```diff
@@ -0,0 +1,161 @@
+# Helium Parameter Validation Documentation
+
+This goals of this document is to define valid/invalid API and Query parameters and provide specifications on resulting behavior of the Helium API.
+
+## Movies
+
+### API /api/movies
+
+### Query Parameters
+
+* Name : q
+* Description : query
+* Type: string
+* Parameter Validation:
+    * Valid input : q length >= 3 characters
```

Comment:
Still need to update the unordered lists throughout the md file to use "-" instead of "*" - my bad on clarity of the previous comment :)